ushahidi/platform-client-mzima
# [Outreachy Task Submission]: Fix collection list responsiveness issue on add post to collection

### Description:
This pull request aims to address the responsiveness issue: ushahidi/platform#4893, observed in the collection list when adding posts. Currently, the collection cards exhibit inconsistency in layout, particularly when the window is resized to a tablet size. 

### Testing Checklist:
1. Launch the web application using `npm run web:serve`.
2. Log in to the application.
3. Click on the "data view" icon on the sidebar.
4. Within the list of posts, click on the more icon(3 dots) and then click on the add to collection icon.
5. Resize the window to a tablet size.
6. Verify that the collection cards are visually consistent and responsive.

### Additional Context:
For more clarification, a video attachment has been provided on the issue [#4893](https://github.com/ushahidi/platform/issues/4893).
